### PR TITLE
Portal updates

### PR DESCRIPTION
**Pull request by subfuzion**
This PR is for @freignat91 portal updates to be merged for the 0.11.x milestone.

### CODE REVIEW

**Comment by subfuzion** [comment]
Closing for now (@freignat is working on a cleaner rebase).

**Comment by freignat91** [comment]
This branch is now clean and ready

**Review by freignat91 — APPROVED** [review]

File: gtw.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      default:
+        aliases:
+          - amplifier-gateway
+    labels:
+      io.amp.role: "infrastructure"
+    environment:
+      - "SERVICE_PORTS=80"
+      - "VIRTUAL_HOST=https://gw.*"
+    ports:
+      - "8080:80"
```
Comment (ndegory) [inline gtw.yml:13]: do you need to expose on port 8080?

File: gtw.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    image: appcelerator/amplifier-gateway:local
+    networks:
+      default:
+        aliases:
+          - amplifier-gateway
+    labels:
+      io.amp.role: "infrastructure"
+    environment:
+      - "SERVICE_PORTS=80"
+      - "VIRTUAL_HOST=https://gw.*"
```
Comment (ndegory) [inline gtw.yml:11]: you don't need redirection from http to https?

File: api/rpc/stats/stats.proto [hunk 3]
```diff
@@ -25,10 +26,11 @@ message MetricsEntry {
   string node_id = 10;
   map<string, string> labels = 11;
   string group = 12;
-  MetricsCPUEntry cpu = 13;
-  MetricsIOEntry io = 14;
-  MetricsMemEntry mem = 15;
-  MetricsNetEntry net = 16;
+  string sgroup = 13;
```
Comment (ndegory) [inline api/rpc/stats/stats.proto:29]: shouldn't we add new fields at the end, instead of breaking the ids?

File: gtw.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    ports:
+      - "8080:80"
+    deploy:
```
Comment (ndegory) [inline gtw.yml:14]: we should add a placement constraint to restrict it on the worker nodes (see other stack files)

File: pkg/docker/docker.go [hunk 5]
```diff
@@ -155,6 +158,7 @@ func (d *Docker) StackList(ctx context.Context) (output string, err error) {
 	return string(output), nil
 }
 
+// StackRemove remoce a stack
```
Comment (ndegory) [inline pkg/docker/docker.go:161]: s/remoce/remove/

File: platform/stacks/portal.stack.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+networks:
+  default:
+    external:
+      name: ampnet
+  infrastructure:
+
+services:
+
+  portal:
+    image: appcelerator/amp-ui:1.0.0
```
Comment (ndegory) [inline platform/stacks/portal.stack.yml:12]: we started aligning the versions of the artifacts. Please use the "${TAG:-0.10.1}" version as for other core services.

File: platform/stacks/portal.stack.yml [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    networks:
+      - default
+    deploy:
```
Comment (ndegory) [inline platform/stacks/portal.stack.yml:15]: should contain a placement constraint too

**Review by ndegory — COMMENTED** [review]
Please also remove portal/.DS_Store, it contains a binary

**Pull request merged**